### PR TITLE
chore: add blog global authors

### PR DESCRIPTION
**Pull request by dsanders11**
Makes use of the [blog global authors feature](https://docusaurus.io/docs/blog#global-authors) in Docusaurus to DRY up our blog posts and simplify setting authors for future posts.

### CODE REVIEW

File: blog/kap.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ authors:
   - name: sindresorhus
     url: 'https://github.com/sindresorhus'
     image_url: 'https://github.com/sindresorhus.png?size=96'
```
Comment (BlackHole1) [inline blog/kap.md:10]: Forgot to modify this.

File: blog/voltra.md [hunk 2]
```diff
@@ -8,9 +8,7 @@ authors:
   - name: aprileelcich
     url: 'https://github.com/aprileelcich'
     image_url: 'https://github.com/aprileelcich.png?size=96'
```
Comment (BlackHole1) [inline blog/voltra.md:10]: Same as above

File: blog/webtorrent.md [hunk 3]
```diff
@@ -5,9 +5,7 @@ authors:
   - name: feross
     url: 'https://github.com/feross'
     image_url: 'https://github.com/feross.png?size=96'
```
Comment (BlackHole1) [inline blog/webtorrent.md:7]: Same as above

File: blog/wordpress.md [hunk 4]
```diff
@@ -8,9 +8,7 @@ authors:
   - name: johngodley
     url: 'https://github.com/johngodley'
     image_url: 'https://github.com/johngodley.png?size=96'
```
Comment (BlackHole1) [inline blog/wordpress.md:10]: Same as above

**Review by BlackHole1 — CHANGES_REQUESTED** [review]
Are these intentionally ignored?

**Comment by dsanders11** [comment]
> Are these intentionally ignored?

I left some authors who are no longer active in the project/had one blog post out of the global authors to keep it a bit smaller. The main benefit of the global authors file is to make setting authors going forward easier, so I focused on active authors and a few who had a number of posts in the past (just to DRY up more files while I was at it).

**Review by erickzhao — APPROVED** [review]
LGTM!

**Pull request merged**